packages/components/Menu/src/MenuTrigger/getRevisedProps.ts

```diff
@@ -0,0 +1,48 @@
+import { AccessibilityActionEvent } from 'react-native';
+import { memoize } from '@fluentui-react-native/framework';
+import { InteractionEvent, isMouseEvent } from '@fluentui-react-native/interactive-hooks';
+import { MenuTriggerChildProps, MenuTriggerState } from './MenuTrigger.types';
+
+export const getRevisedProps = memoize(getRevisedPropsWorker);
+function getRevisedPropsWorker(state: MenuTriggerState, props: any): MenuTriggerChildProps {
+  const revisedProps = { ...state.props };
+  if (props.accessibilityState) {
+    revisedProps.accessibilityState = { ...state.props.accessibilityState, ...props.accessibilityState };
+  }
+
+  if (props.accessibilityActions) {
+    revisedProps.accessibilityActions = [...state.props.accessibilityActions, ...props.accessibilityActions];
+  }
+
+  if (props.onAccessibilityAction) {
+    revisedProps.onAccessibilityAction = (e: AccessibilityActionEvent) => {
+      props.onAccessibilityAction(e);
+      state.props.onAccessibilityAction(e);
+    };
+  }
+
+  if (props.onClick) {
+    revisedProps.onClick = (e: InteractionEvent) => {
+      props.onClick(e);
+      state.props.onClick(e);
+    };
+  }
+
+  let onHoverIn = undefined;
```

Comment:
This is done because onHoverIn and onHoverOut are Readonly properties, so I have to explicitly add them through creating a new object below instead of assigning them to revisedProps directly.

Reply:
Post merge question: What's making them readonly? Should we fix a type somewhere to make it not? Was that because of my last merge with all the Pressable stuff?

Reply:
Not anymore because MenuV1 doesn't use our fork anymore?

Reply:
I think we (APEX) will be looking at MenuV1 again relatively soon, so I'll take a second look myself when we do.

Reply:
It's because the MenuTriggerChildProps extends from IWithPressableOptions which uses the type I linked above. If it should derive from something else, then yeah that should be fixed.

Reply:
I guess I missed that. That should be `PressablePropsExtended` No worries, I'll take care of that when I touch these myself.